content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
+{{</todo>}}
 
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
+### Processing Assets for Multiple Platforms
+
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.
+
+### Asset Job Management
+
+By default, the Asset Processor will create a worker job for each logical core on the host machine. If you wish to manage this yourself, you can adjust the setting `Amazon/AssetProcessor/Settings/Jobs/maxJobs`. A value of zero will have the Asset Processor create a job for each logical core on your machine. A value of one can be useful when debugging, if you wish to attach a debugger to a running builder processing a specific asset. Other values lower than your logical core count can be useful to manage the impact asset processing has on your machine while you are trying to perform other tasks, with the tradeoff of a longer time before all assets are processed.
+
+### Asset Cache Server
+
+The Asset Cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. In most cases this will be a network share, an example for Windows looks like `T:\\AssetServerCache`. This feature is in development. If you have questions about the Asset Cache server, reach out to us on the [O3DE Discord](https://discord.com/invite/o3de).
+
+### Metadata Files
 
-* Add scan directories for Asset Processor.
+Metadata files for assets are managed under the setting path `Amazon/AssetProcessor/Settings/MetaDataTypes`, with a format mapping the metadata extension to associated file extension. For example, FBX files use a metadata file type with the extension `.fbx.assetinfo`, which is defined under this setting path as `"fbx.assetinfo": "fbx"`. When adding a new asset type and want to track settings in a metadata file, make sure to add it to this setting path. This is so the Asset Processor will re-run the processing job for the source asset when the metadata file changes.
+
+### Scan Folders
 
-* Set the number of process jobs.
-* Specify *metafile fingerprinting*, that is, which files trigger related files to be rebuilt.
+Scan folders can be added as registry settings. In most cases you'll use the asset folders of Gems or your project, which will be automatically added as scan folders. However, if you need to add a new scan folder for tracking source assets, you can add it under the registry setting path `Amazon/AssetProcessor/Settings/` with a pattern of:

```

Comment:
For continuity:
```suggestion
Scan folders can be added as registry settings. In most cases you'll use the asset folders of Gems or your project, which will be automatically added as scan folders. However, if you need to add a new scan folder for tracking source assets, you can add it under the registry setting path `Amazon/AssetProcessor/Settings/ScanFolder` with a pattern of:
```

Reply:
Okay, I guess it depends on how you define "under". In that case, Exclude should be under `Amazon/AssetProcessor/Settings`, not `Amazon/AssetProcessor/Settings/Exclude`.